arch/arm64/Kconfig

```diff
@@ -97,6 +97,7 @@ config ARCH_CHIP_IMX9
 	select ARCH_HAVE_I2CRESET
 	select ARCH_HAVE_IRQTRIGGER
 	select ARCH_NEED_ADDRENV_MAPPING
+	select LIBC_ARCH_ELF_64BIT if LIBC_ARCH_ELF
```

Comment:
should we move to ARCH_ARM64? @pussuw ?

Reply:
Do you mean here ?

https://github.com/apache/nuttx/blob/575c9b12ab1c809d00600bb51ec9ddb79fba1b1c/arch/Kconfig#L26-L42

Reply:
Yes, and remove `select LIBC_ARCH_ELF_64BIT if LIBC_ARCH_ELF` from individual chip